4-star-nosed_mole/api_definitions/rest/drs3.yaml

```
@@ -0,0 +1,225 @@
+        self_uri:
+          type: string
+          description: |-
+            A drs:// hostname-based URI, as defined in the DRS documentation, that tells clients how to access this object.
+            The intent of this field is to make DRS objects self-contained, and therefore easier for clients to store and pass around.  For example, if you arrive at this DRS JSON by resolving a compact identifier-based DRS URI, the `self_uri` presents you with a hostname and properly encoded DRS ID for use in subsequent `access` endpoint calls.
+          example: drs://drs.example.org/314159
+        size:
+          type: integer
+          description: |-
+            For blobs, the blob size in bytes.
+            For bundles, the cumulative size, in bytes, of items in the `contents` field.
+          format: int64
+        created_time:
+          type: string
+          description: |-
+            Timestamp of content creation in RFC3339.
+            (This is the creation time of the underlying content, not of the JSON object.)
+          format: date-time
+        checksums:
+          minItems: 1
+          type: array
+          description:
+            "The checksum of the `DrsObject`. At least one checksum must
+            be provided. For blobs, the checksum is computed over the bytes in the
+            blob. For bundles, the checksum is computed over a sorted concatenation
+            of the checksums of its top-level contained objects (not recursive, names
+            not included). The list of checksums is sorted alphabetically (hex-code)
+            before concatenation and a further checksum is performed on the concatenated
+            checksum value. For example, if a bundle contains blobs with the following
+            checksums: md5(blob1) = 72794b6d md5(blob2) = 5e089d29 Then the checksum
+            of the bundle is: md5( concat( sort( md5(blob1), md5(blob2) ) ) ) = md5(
+            concat( sort( 72794b6d, 5e089d29 ) ) ) = md5( concat( 5e089d29, 72794b6d
+            ) ) = md5( 5e089d2972794b6d ) = f7a29a04"
+          items:
+            $ref: "#/components/schemas/Checksum"
+        access_methods:
+          minItems: 1
+          type: array
+          description: |-
+            The list of access methods that can be used to fetch the `DrsObject`.
+            Required for single blobs; optional for bundles.
+          items:
+            $ref: "#/components/schemas/AccessMethod"
+    AccessURL:
+      required:
+        - url
+        - headers
+      type: object
+      properties:
+        url:
+          type: string
+          description:
+            A fully resolvable URL that can be used to fetch the actual
+            object bytes.
+        headers:
+          type: array
+          description:
+            An optional list of headers to include in the HTTP request
```

```suggestion
            A list of headers to include in the HTTP request
```
As it is required in our case